src/nxdoc/nuxeo-server/upgrading-the-nuxeo-platform/upgrade-from-lts-2016-following-fast-tracks.md

```diff
@@ -400,3 +400,87 @@ Nuxeo Segment.io plugin is now aligned on `com.github.segmentio:analytics:1.0.7`
 - [Release notes for 9.2]({{page version='' space='nxdoc' page='nuxeo-server-release-notes'}})
 
 {{! /multiexcerpt}}
+
+
+{{! multiexcerpt name='9.2-to-9.3-upgrade-page'}}
+## From 9.2 to 9.3
+
+### Installation and Configuration
+
+{{! multiexcerpt name='upgrade-8.10-installation-elasticsearch-upgrade'}}
+#### Elasticsearch Upgrade
+
+A new version of Elasticsearch is required, here are the necessary steps to upgrade:
+- Upgrade Elasticsearch version
+- Update your custom settings and mapping to follow new Elasticsearch rules
+- Re-index the repository to apply the new settings and mapping
+- Update your custom code that query Elasticsearch directly
+
+** Upgrade Elasticsearch Version **
+
+Upgrading Elasticsearch to 5.6.x is required.
+
+In order to upgrade your cluster to 5.6.x, please follow the [Elasticsearch documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/setup-upgrade.html).
+This will requires a [Full cluster restart upgrade](https://www.elastic.co/guide/en/elasticsearch/reference/current/restart-upgrade.html).
+
+** Update your custom Elasticsearch settings and mapping **
+
+If you have a custom configuration for Elasticsearch it needs to be adapted to follow the new 5.6 rules.
+It is probably easier to start from the default Nuxeo 9.3 configuration and migrate your necessary change instead of modifying directly your existing contribution.
```

Comment:
"modifying directly your existing contribution" to "modifying your existing contribution directly"